config.json

```diff
@@ -802,6 +802,17 @@
       ]
     },
     {
+      "slug": "two-fer",
```

Comment:
Could please add `uuid` since it's required for every exercise?
It can be generated with `configlet` - https://github.com/exercism/configlet
```
configlet uuid
```

Here is one I've generated `bacb2184-025f-0980-38d6-34fb419e03a6d3d8b44`